cognite/neat/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.36.0"
+__version__ = "0.36.1"
```

Comment:
@alivinco I think you did not only repair bugs but added new feature, so bump to `0.37.0`